core/src/bin/test_auth_flow.rs

```diff
@@ -1,17 +1,21 @@
-use mongo_odbc_core::oidc_auth::*;
+use mongo_odbc_core::oidc_auth::do_auth_flow;
+use mongodb::options::oidc::{CallbackContext, IdpServerInfo};
 
 #[tokio::main]
 async fn main() {
```

Comment:
The actual public interface for the driver requires using a builder in case the types change in the future. This is the community way to do structs in public interfaces